exercises/resistor-significants/description.md

```diff
@@ -0,0 +1,16 @@
+If you would build anything with a Raspberry Pi or an Arduino, one of the first components you need are resistors. In this exercise, it doesn't matter what a resistor is or does, the only thing that matters is that they come with different values. And that they are tiny. So tiny, that it's hard to read such a value if it were printed on the thing. To solve that, fabricators use color codes that reveal their value. Which is great, but it makes it hard to see the actual value.
```

Comment:
I have some thoughts on rewording this but I don't have the time at the moment.  This evening I'll revisit this and update this comment...

Reply:
I wasn't paying very careful attention to what comes after what I just wrote.  Some of it may be redundant.

Reply:
@F3PiX Could you look at the suggested changes?

Reply:
Thanks @rpottsoh ! 
It's not so easy to distinguish the separate changes in texts with the Suggested Changes feature.  What I see, there's some corrections of mistakes (thanks!). There's also some level of detail added that I'd rather keep out. Let me see how I can work the changes in.